var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -59,13 +56,18 @@ def filter_sbang(self):
                     *files)
 
     def install(self, spec, prefix):
-        if self.compiler.name not in self.supported_compilers:
+        supported_compilers = {'clang': 'CLANG', 'gcc': 'GCC', 'intel': 'ICC'}
+        if spec.target() == 'aarch64':
```

Comment:
Hmm, did you try installing this recipe? `spec.target` is supposed to be a property see [docs](https://spack.readthedocs.io/en/latest/packaging_guide.html#querying-the-target-microarchitecture):
```python
Spack version 0.14.0
Python 3.7.4, Linux x86_64
>>> import spack.spec
>>> s = spack.spec.Spec('likwid')
>>> s.concretize()
>>> str(s.target)
'broadwell'
>>> str(s.target())
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: 'Microarchitecture' object is not callable
```

Reply:
```suggestion
        if spec.target.family == 'aarch64':
```